packages/contracts/contracts/L1/Lib_WithdrawalVerifier.sol

```diff
@@ -0,0 +1,57 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import { L2OutputOracle } from "./L2OutputOracle.sol";
+import {
+    Lib_SecureMerkleTrie
+} from "../../lib/optimism/packages/contracts/contracts/libraries/trie/Lib_SecureMerkleTrie.sol";
+
+/**
+ * @title WithdrawalVerifier
+ */
+library WithdrawalVerifier {
+    struct OutputRootProof {
+        uint256 timestamp;
```

Comment:
Why is the timestamp on this struct if it is unused?

Reply:
Good callout, removed.